saved_chart_example.html

```diff
@@ -0,0 +1,675 @@
+
```

Comment:
Why is this file updated?

Reply:
This wasn't supposed to be updated. Not until the pull request I noticed this file was updated. I thought there would be some version numbers and metadata updates. This would have happened possibly after I installed new versions of python packages. I thought the changes could be important, and tests didn't fail too, so I didn't doubt the change much.

Reply:
The changes I made on this were just small ones. I thought it would be a good first issue.

Reply:
Ok, I see, thanks! I included to change in #147.